testnet/src/tests/integrations.rs

```diff
@@ -747,8 +747,9 @@ fn bad_contract_tx_rollback() {
             let (burn_header_hash, block_hash) = (&tenure.parent_block.metadata.burn_header_hash, &tenure.parent_block.metadata.anchored_header.block_hash());
             tenure.mem_pool.submit_raw(burn_header_hash, block_hash, xfer_to_contract).unwrap();
             
-            let xfer_to_contract = make_stacks_transfer(&sk_3, 2, 0, &addr_2.into(), 3000);
-            tenure.mem_pool.submit_raw(burn_header_hash, block_hash, xfer_to_contract).unwrap();
+            // doesn't consistently get mined by the StacksBlockBuilder, because order matters!
+            // let xfer_to_contract = make_stacks_transfer(&sk_3, 2, 0, &addr_2.into(), 3000);
+            // tenure.mem_pool.submit_raw(burn_header_hash, block_hash, xfer_to_contract).unwrap();
```

Comment:
Should we introduce an additional round, so we can keep this test?

Reply:
Sure -- will add.

Reply:
Added in 53b119a